dal/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ dependencies {
     jooqGeneratorRuntime(group = "org.jooq", name = "jooq-codegen", version = jooqVersion)
     jooqGeneratorRuntime("org.postgresql:postgresql:42.2.2")
 }
+tasks["build"].dependsOn(":db:flywayMigrate")
```

Comment:
I'd suggest to make a most specific task depend on `flywayMigrate`: `jooqGenerate`